src/openrct2/interface/Window.cpp

```diff
@@ -328,6 +328,7 @@ WindowBase* WindowFindByNumber(WindowClass cls, EntityId id)
 void WindowCloseTop()
 {
     WindowCloseByClass(WindowClass::Dropdown);
+    WindowCloseByClass(WindowClass::Loadsave);
```

Comment:
This will _always_ close the Load/Save window, even when it wasn’t the one on top.